server/utils/placementRequests/checkYourAnswersUtils.ts

```diff
@@ -19,9 +18,10 @@ export const getPageTitle = (placementApplication: PlacementApplication, pageNam
   return new Page(placementApplication.data?.['request-a-placement'][pageName], placementApplication).title
 }
 
-export const pagesForReview = () => {
-  const task = PlacementRequest.sections.map(section => section.tasks).flat()[0]
-  return Object.keys(task.pages).filter(page => page !== 'check-your-answers')
+export const pagesForReview = (placementApplication: PlacementApplication) => {
```

Comment:
Bryan made a really good point the other day - if someone goes in and fills out a question, then edits their response to something that alters the branching, the previous response will still show, so we really should be walking the tree to make sure we only return relevant responses, and filter the other ones out. Not for now though, just to be aware.

Reply:
Yeah I saw that, great perception.  
Hopefully it won't bite us in the short term